editoast/src/views/authz.rs

```diff
@@ -132,13 +130,68 @@ async fn whoami(Extension(auth): AuthenticationExt) -> Result<Json<WhoamiRespons
     }))
 }
 
+#[derive(Debug, Serialize, ToSchema)]
+#[cfg_attr(test, derive(Deserialize, PartialEq, Eq))]
+struct ResourcePrivileges {
+    id: i64,
```

Comment:
nitpick: not being very familiar with this functionality about privileges, I was not sure about what to expect from the response. And reading `ResourcePrivileges`, I actually wondered what this `id` was: is it a User ID, a privilege ID (what that would mean? I'm not sure), a resource ID (it's this one, if I understand the rest of the code). Do you think it'd be better to be more explicit here?

```suggestion
    resource_id: i64,
```

Reply:
We thought there would be no mistake about these ids referring resources in the context (there is no such thing as privilege ids). But then again, explicit better than implicit, especially since you left a comment. @sim51 ok for the front-end part?

Reply:
yes, no prob

Reply:
<details>

```diff
diff --git a/editoast/openapi.yaml b/editoast/openapi.yaml
index 9f10447e9..1f73b4eb3 100644
--- a/editoast/openapi.yaml
+++ b/editoast/openapi.yaml
@@ -150,17 +150,17 @@ paths:
                   items:
                     type: object
                     required:
-                    - id
+                    - resource_id
                     - privileges
                     properties:
-                      id:
-                        type: integer
-                        format: int64
                       privileges:
                         type: array
                         items:
                           $ref: '#/components/schemas/InfraPrivilege'
                         uniqueItems: true
+                      resource_id:
+                        type: integer
+                        format: int64
   /authz/{resource_type}/{resource_id}:
     get:
       tags:
diff --git a/editoast/src/views/authz.rs b/editoast/src/views/authz.rs
index 4d060514c..0c02541b3 100644
--- a/editoast/src/views/authz.rs
+++ b/editoast/src/views/authz.rs
@@ -149,7 +149,7 @@ async fn whoami(Extension(auth): AuthenticationExt) -> Result<Json<WhoamiRespons
 #[derive(Debug, Serialize, ToSchema)]
 #[cfg_attr(test, derive(Deserialize, PartialEq, Eq))]
 struct ResourcePrivileges {
-    id: i64,
+    resource_id: i64,
     privileges: HashSet<InfraPrivilege>,
 }
 
@@ -188,7 +188,7 @@ async fn user_privileges(
                         .entry(ResourceType::Infra)
                         .or_default()
                         .push(ResourcePrivileges {
-                            id: resource_id,
+                            resource_id,
                             privileges: authorizer
                                 .infra_privileges(&authz::Infra(resource_id))
                                 .await
@@ -581,7 +581,12 @@ mod tests {
             .remove(&ResourceType::Infra)
             .unwrap()
             .into_iter()
-            .map(|ResourcePrivileges { id, privileges }| (id, privileges))
+            .map(
+                |ResourcePrivileges {
+                     resource_id,
+                     privileges,
+                 }| (resource_id, privileges),
+            )
             .collect::<HashMap<_, _>>();
         assert_eq!(
             privileges.remove(&infra1).unwrap(),
@@ -642,7 +647,12 @@ mod tests {
             .remove(&ResourceType::Infra)
             .unwrap()
             .into_iter()
-            .map(|ResourcePrivileges { id, privileges }| (id, privileges))
+            .map(
+                |ResourcePrivileges {
+                     resource_id,
+                     privileges,
+                 }| (resource_id, privileges),
+            )
             .collect::<HashMap<_, _>>();
         assert_eq!(
             privileges.remove(&infra1).unwrap(),
diff --git a/front/src/common/api/generatedEditoastApi.ts b/front/src/common/api/generatedEditoastApi.ts
index 9af2d436f..9026e425b 100644
--- a/front/src/common/api/generatedEditoastApi.ts
+++ b/front/src/common/api/generatedEditoastApi.ts
@@ -1282,8 +1282,8 @@ export type PostAuthzMeGrantsApiArg = {
 export type PostAuthzMePrivilegesApiResponse =
   /** status 200 The privileges of the user sending the request over each requested resource. */ {
     [key: string]: {
-      id: number;
       privileges: InfraPrivilege[];
+      resource_id: number;
     }[];
   };
 export type PostAuthzMePrivilegesApiArg = {
```

</details>